youtube_dl/extractor/brightcove.py

```diff
@@ -489,34 +491,56 @@ def _extract_url(webpage):
     @staticmethod
     def _extract_urls(webpage):
         # Reference:
-        # 1. http://docs.brightcove.com/en/video-cloud/brightcove-player/guides/publish-video.html#setvideoiniframe
+        # 1. http://docs.brightcove.com/en/video-cloud/brightcove-player/guides/publish-video.html#tag
         # 2. http://docs.brightcove.com/en/video-cloud/brightcove-player/guides/publish-video.html#setvideousingjavascript
-        # 3. http://docs.brightcove.com/en/video-cloud/brightcove-player/guides/embed-in-page.html
-        # 4. https://support.brightcove.com/en/video-cloud/docs/dynamically-assigning-videos-player
+        # 3. http://docs.brightcove.com/en/video-cloud/brightcove-player/guides/publish-video.html#setvideoiniframe
```

Comment:
I guess so…the code's not now so linear, since we've merged cases as Yen requested, so it doesn't quite go back that way.

Do you want me to put them back in the older order anyhow? Happy to, but it does seem more readable now than it was.